### ISSUE TITLE
Ability to pop out video tiles ("picture-in-picture")

### ISSUE THREAD

**Issue by SimonBrandner** [vector-im/element-call#1229]
One request that we've been getting recently is a way to activate the browser's [picture-in-picture functionality](https://developer.mozilla.org/en-US/docs/Web/API/Picture-in-Picture_API) from within Element Call.

Yet to be discovered:
- What are the use cases for this?
- Would we rather provide a feature that lets you pop out a specific tile only, or one that creates a window always highlighting the active speaker?
- On the implementation side: how does this interact with LiveKit automatically unsubscribing from video streams that are off screen?
- Is this worth implementing given that Firefox doesn't yet support the relevant APIs?
- Design (potential for integrating it with the drag-and-drop interaction, such that you could literally drag a tile and drop it outside the window to pop it out, much like a browser tab? or do we need something less surprising like a button)

**Comment by robintown** [comment]
I've elaborated on this ticket based on what I believe the title was referring to

**Comment by bblacher** [comment]
I'd love to have the ability to pop out specific screen shares. This would mean that one can watch two screen shares at the same time. (This ability should be in element desktop as well)
I think discord does a great job at this.

**Comment by theraspb3rry** [comment]
Yep! Agreed with @bblacher. My friends and I often have multiple screen shares going at the same time and one of the things you can do on discord is pop out the entire video section and also tile all the video and screen share streams. Even if its a hidden feature enabled in the settings this would be ok for our use case.

Use-cases:
- Multiple views of people playing the same game (we play Escape From Tarkov which this situation comes up often)
- People working on a common goal such as programming or debugging an issue live
- Casually watching a whole bunch of friends play different games
- cause you can

I'd imagine it work similar to discord in a way, create a new window and either select unique streams or show all video streams. I think the "double click" to expand UX in discords window is very intuitive in general (not really specific to this issue), but getting it to only show what you want can be a little annoying. I can further elaborate with pictures and more detailed instructions if its helpful.

**Comment by rriemann** [comment]
I think Zoom has it and the picture stays on top of other applications. That's what people know from Webex and Teams. So for feature parity in corporate settings, very relevant in my view.

Super helpful when multi tasking during a call.
